src/components/Reports/AdditionalSalaryRequest/AdditionalSalaryRequest.tsx

```diff
@@ -1,28 +1,43 @@
 import React from 'react';
-import { Typography } from '@mui/material';
 import { useTranslation } from 'react-i18next';
 import { useAccountListId } from 'src/hooks/useAccountListId';
 import { PanelLayout } from '../Shared/CalculationReports/PanelLayout/PanelLayout';
 import { useIconPanelItems } from '../Shared/CalculationReports/PanelLayout/useIconPanelItems';
+import { Receipt } from '../Shared/CalculationReports/ReceiptStep/Receipt';
 import { PanelTypeEnum } from '../Shared/CalculationReports/Shared/sharedTypes';
 import { StepsList } from '../Shared/CalculationReports/StepsList/StepsList';
 import { AboutForm } from './AboutForm/AboutForm';
 import { AdditionalSalaryRequestSectionEnum } from './AdditionalSalaryRequestHelper';
 import { CompleteForm } from './CompleteForm/CompleteForm';
 import { useAdditionalSalaryRequest } from './Shared/AdditionalSalaryRequestContext';
+import { AdditionalSalaryRequestSection } from './SharedComponents/AdditionalSalaryRequestSection';
 import { SectionPage } from './SharedComponents/SectionPage';
 
 export const MainContent: React.FC = () => {
   const { currentStep } = useAdditionalSalaryRequest();
   const { t } = useTranslation();
+  const accountListId = useAccountListId();
+
+  const viewLink = `/accountLists/${accountListId}/reports/additionalSalaryRequest/view`;
+  const editLink = `/accountLists/${accountListId}/reports/additionalSalaryRequest/edit`;
```

Comment:
Can we have 1 page instead of 2 for the view and edit? unless they are one page and we just change how we show data base on the last URL parameter